Automattic/newspack-plugin
# Feat/add media partners

### All Submissions:






### Changes proposed in this Pull Request:



Adds the https://github.com/Automattic/newspack-media-partners plugin as an optional module. Similar to the RSS Enhancements plugin story (https://github.com/Automattic/newspack-plugin/pull/1688), the module will replace the plugin if the plugin is active. 

This also adds few new features to the Media Partners taxonomy (see 1200133283036252-as-1205658718219681):

1.  Ability to remove the media partner logo
2. Ability to modify the "This story also appeared in" text
3. Ability to skip the media partner markup in RSS feeds (a global setting) 

Note: https://github.com/Automattic/newspack-media-partners should be archived after this is released

### How to test the changes in this Pull Request:

1. Install and activate the [`newspack-media-partners` plugin](https://github.com/Automattic/newspack-media-partners) 
5. Load the admin page, go to Posts -> Media Partners 
6. Observe that the plugin has been deactivated
7. Create a media partner with an image and assign it to a post (it's a taxonomy)
8. Visit the post – observe the attribution message close to the beginning of the post 
9. Edit the media partner, remove the image and change the attribution message 
10. Verify this is updated in the post
11. Visit your site's `/feed` and observe the media partner markup is there
12. Toggle the "Skip in RSS feeds" setting, observe that now the markup does not appear in the feed

### Other information:
